README.md

```diff
@@ -363,7 +363,7 @@ for a curated list of awesome apps/libraries built with `ratatui`!
 
 ## Alternatives
 
-You might want to checkout [Cursive](https://github.com/gyscos/Cursive) or 
```

Comment:
Can you revert this formatting change? It seems unrelated :)

Reply:
I'm fine with this - the EOL space doesn't belong.

Reply:
I think this PR is not relevant to this part. It's my fault by using auto-formatter.